packages/@aws-cdk/aws-rds/lib/endpoint.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * Connection endpoint of a database cluster or instance
+ *
+ * Consists of a combination of hostname and port.
+ */
+export class Endpoint {
+  /**
+   * The hostname of the endpoint
+   */
+  public readonly hostname: string;
+
+  /**
+   * The port of the endpoint
+   */
+  public readonly port: string;
+
+  /**
+   * The combination of "HOSTNAME:PORT" for this endpoint
+   */
+  public readonly socketAddress: string;
```

Comment:
I think I'd rather this is called `address`.

Reply:
This comes from the original implementation for the cluster construct (I just extracted to a separate file). Changing this will be breaking, ok with this?